packages/smithy-http/src/smithy_http/aio/protocols.py

```diff
@@ -116,3 +137,69 @@ async def deserialize_response[
         )
 
         return operation.output.deserialize(deserializer)
+
+    def _is_success(
+        self,
+        operation: APIOperation[Any, Any],
+        context: TypedProperties,
+        response: HTTPResponse,
+    ) -> bool:
+        return 200 <= response.status <= 299
```

Comment:
nit; this is mostly for consistency with other tooling. 299 isn't something you generally think of with http status codes.
```suggestion
        return 200 <= response.status < 300
```